README.md

```diff
@@ -15,12 +15,12 @@ Options:
   --config-file PATH  Path to the config file  [required]
   --days INTEGER      Since how many days ago to backup tickets
   --help              Show this message and exit.
-  
+
 ```
-  
+
 #### Dependencies
 * zendesk
-* click 
+* click
```

Comment:
might be that you did not pool the latest version from this repo.... just guessing....